packages/content/src/Application/PropertyResolver/Resolver/BlockPropertyResolver.php

```diff
@@ -94,6 +94,13 @@ public function resolve(mixed $data, string $locale, array $params = []): Conten
             );
         }
 
+        $minOccurs = $metadata->getMinOccurs();
+        $maxOccurs = $metadata->getMaxOccurs();
+
+        if (1 === $minOccurs && 1 === $maxOccurs && \count($contentViews) > 0) {
+            $contentViews = $contentViews[0]->getContent();
```

Comment:
@Prokyonn aren't we loosing the `view` now?

Reply:
You were right, fixed it.